exec/network_drop.go

```diff
@@ -92,31 +96,40 @@ func (ne *NetworkDropExecutor) Exec(suid string, ctx context.Context, model *spe
 	}
 	localPort := model.ActionFlags["local-port"]
 	remotePort := model.ActionFlags["remote-port"]
+	urlPattern := model.ActionFlags["url-pattern"]
 	if _, ok := spec.IsDestroy(ctx); ok {
-		return ne.stop(localPort, remotePort, ctx)
+		return ne.stop(localPort, remotePort, urlPattern, ctx)
 	}
 
-	return ne.start(localPort, remotePort, ctx)
+	return ne.start(localPort, remotePort, urlPattern, ctx)
 }
 
-func (ne *NetworkDropExecutor) start(localPort, remotePort string, ctx context.Context) *spec.Response {
+func (ne *NetworkDropExecutor) start(localPort, remotePort, urlPattern string, ctx context.Context) *spec.Response {
 	args := fmt.Sprintf("--start --debug=%t", util.Debug)
 	if localPort != "" {
 		args = fmt.Sprintf("%s --local-port %s", args, localPort)
 	}
-	if remotePort != "" {
-		args = fmt.Sprintf("%s --remote-port %s", args, remotePort)
+	if urlPattern != "" {
+		args = fmt.Sprintf("%s --remote-port %s -m string --string %s --algo bm", args, remotePort, urlPattern)
```

Comment:
It doesn't work here when I use the balde cli command, you can write it like this
````
if remotePort != "" {
		args = fmt.Sprintf("%s --remote-port %s", args, remotePort)
}
if urlPattern != "" {
		args = fmt.Sprintf("%s --url-pattern %s", args, urlPattern)
}
````
````
blade create network drop --remote-port 80 --url-pattern publish
````
{"code":604,"success":false,"error":"flag provided but not defined: -m\nUsage of /root/chaosblade/target/chaosblade-0.7.0/bin/chaos_dropnetwork:\n  -debug\n    \tset debug mode\n  -local-port string\n    \tlocal port\n  -remote-port string\n    \tremote port\n  -start\n    \tstart drop\n  -stop\n    \tstop drop\n  -url-pattern string\n    \turl pattern\n exit status 2"}

Reply:
it does not work after modify as you said, is there something i missed？